appengine/genetics/js/blocks.js

```diff
@@ -291,7 +293,7 @@ Blockly.Blocks['genetics_getProperties'] = {
    * @this Blockly.Block
    */
   updateType_: function(newProp) {
-    if (newProp == 'SEX') {
+    if (newProp === 'SEX') {
```

Comment:
Most other companies standardize on === for normal compares, Google does the opposite using == for normal compares.  Thus at Google === is an indication that there's a strange comparison happening where null, undefined or some other falsey thing is involved.

Here and below, do a search for: ===